src/tests/unitTests/tripService.test.js

```diff
@@ -59,3 +60,16 @@ describe('APPROVE REQUEST SERVICE TEST', () => {
     expect(approvedRequest.status).to.equal('approved');
   });
 });
+describe('MOST TRAVELLED DESTINATION UNIT TEST', () => {
+  it('should return the most travelled destinations', async () => {
+    const counting = await db.Request.findAll({
```

Comment:
Great work adding unit tests!! can you add integration tests.

Reply:
They are added successfully.